solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -67,7 +65,7 @@
 // servlets that are more focused in scope. This should become possible now that we have a
 // ServletContextListener for startup/shutdown of CoreContainer that sets up a service from which
 // things like CoreContainer can be requested. (or better yet injected)
-public class SolrDispatchFilter extends HttpFilter implements PathExcluder {
```

Comment:
looks like you can also remove PathExcluder

Reply:
and pull methods out of ServletUtils too! this is fun :-)